examples/django/django_exampleapp/README.md

```diff
@@ -0,0 +1,36 @@
+# Django Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
+```
```

Comment:
If the user is expected to replace the default `project_id` and `project_key` values in [`examples/django/django_exampleapp/django_exampleapp/settings.py`](https://github.com/airbrake/pybrake/pull/177/files#diff-543d7436e08bb0cef2501c0c0138906c62bd46db02e01a43e035ecd56055070bR31-R34), then we should add that as a step here right?

Reply:
Hey @thompiler, added the suggested point in README file. I will get a confirmation about your other suggestion as well and update it.